dotcom-rendering/src/lib/discussionApi.tsx

```diff
@@ -65,7 +69,7 @@ export const getDiscussion = async (
 		threads: ThreadsType;
 		page: number;
 	},
-): Promise<DiscussionResponse | undefined> => {
+): Promise<Result<GetDiscussionError, DiscussionSuccess>> => {
```

Comment:
Small thing, but perhaps `DiscussionSuccess` should be `GetDiscussionSuccess`? There are a few different discussion API calls, so `DiscussionSuccess` may be too generic a term?